### ISSUE TITLE
Prod profile not working

### ISSUE THREAD

**Issue by sonersivri** [jhipster/generator-jhipster#5113]
Prod profile generated war not working 
version 4.0.1

I Delete 'infinite-scroll',
            'angular-loading-bar'
from app/app.module.js and it works

vendor-a935ac298f.js:4Uncaught Error: [$injector:modulerr] Failed to instantiate module ceptecafeApp due to:
Error: [$injector:modulerr] Failed to instantiate module infinite-scroll due to:
Error: [$injector:nomod] Module 'infinite-scroll' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
http://errors.angularjs.org/1.5.8/$injector/nomod?p0=infinite-scroll
    at http://52.23.175.251/app/vendor-a935ac298f.js:4:28233
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:6380
    at n (http://52.23.175.251/app/vendor-a935ac298f.js:5:5853)
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:6165
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15513
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15530
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
http://errors.angularjs.org/1.5.8/$injector/modulerr?p0=infinite-scroll&p1=Error%3A%20%5B%24injector%3Anomod%5D%20Module%20'infinite-scroll'%20is%20not%20available!%20You%20either%20misspelled%20the%20module%20name%20or%20forgot%20to%20load%20it.%20If%20registering%20a%20module%20ensure%20that%20you%20specify%20the%20dependencies%20as%20the%20second%20argument.%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fnomod%3Fp0%3Dinfinite-scroll%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28233%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6380%0A%20%20%20%20at%20n%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A5853)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6165%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15513%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15530%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)
    at http://52.23.175.251/app/vendor-a935ac298f.js:4:28233
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15789
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15530
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at rt (http://52.23.175.251/app/vendor-a935ac298f.js:5:17290)
    at a (http://52.23.175.251/app/vendor-a935ac298f.js:5:3923)
    at ie (http://52.23.175.251/app/vendor-a935ac298f.js:5:4233)
http://errors.angularjs.org/1.5.8/$injector/modulerr?p0=ceptecafeApp&p1=Error%3A%20%5B%24injector%3Amodulerr%5D%20Failed%20to%20instantiate%20module%20infinite-scroll%20due%20to%3A%0AError%3A%20%5B%24injector%3Anomod%5D%20Module%20'infinite-scroll'%20is%20not%20available!%20You%20either%20misspelled%20the%20module%20name%20or%20forgot%20to%20load%20it.%20If%20registering%20a%20module%20ensure%20that%20you%20specify%20the%20dependencies%20as%20the%20second%20argument.%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fnomod%3Fp0%3Dinfinite-scroll%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28233%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6380%0A%20%20%20%20at%20n%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A5853)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6165%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15513%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15530%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fmodulerr%3Fp0%3Dinfinite-scroll%26p1%3DError%253A%2520%255B%2524injector%253Anomod%255D%2520Module%2520'infinite-scroll'%2520is%2520not%2520available!%2520You%2520either%2520misspelled%2520the%2520module%2520name%2520or%2520forgot%2520to%2520load%2520it.%2520If%2520registering%2520a%2520module%2520ensure%2520that%2520you%2520specify%2520the%2520dependencies%2520as%2520the%2520second%2520argument.%250Ahttp%253A%252F%252Ferrors.angularjs.org%252F1.5.8%252F%2524injector%252Fnomod%253Fp0%253Dinfinite-scroll%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A4%253A28233%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A6380%250A%2520%2520%2520%2520at%2520n%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A5853)%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A6165%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15513%250A%2520%2520%2520%2520at%2520r%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A4%253A28681)%250A%2520%2520%2520%2520at%2520f%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15361)%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15530%250A%2520%2520%2520%2520at%2520r%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A4%253A28681)%250A%2520%2520%2520%2520at%2520f%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28233%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15789%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15530%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20rt%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A17290)%0A%20%20%20%20at%20a%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A3923)%0A%20%20%20%20at%20ie%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A4233)
    at http://52.23.175.251/app/vendor-a935ac298f.js:4:28233
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:6380
    at n (http://52.23.175.251/app/vendor-a935ac298f.js:5:5853)
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:6165
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15513
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15530
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
http://errors.angularjs.org/1.5.8/$injector/modulerr?p0=infinite-scroll&p1=Error%3A%20%5B%24injector%3Anomod%5D%20Module%20'infinite-scroll'%20is%20not%20available!%20You%20either%20misspelled%20the%20module%20name%20or%20forgot%20to%20load%20it.%20If%20registering%20a%20module%20ensure%20that%20you%20specify%20the%20dependencies%20as%20the%20second%20argument.%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fnomod%3Fp0%3Dinfinite-scroll%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28233%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6380%0A%20%20%20%20at%20n%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A5853)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6165%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15513%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15530%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)
    at http://52.23.175.251/app/vendor-a935ac298f.js:4:28233
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15789
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15530
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at rt (http://52.23.175.251/app/vendor-a935ac298f.js:5:17290)
    at a (http://52.23.175.251/app/vendor-a935ac298f.js:5:3923)
    at ie (http://52.23.175.251/app/vendor-a935ac298f.js:5:4233)
http://errors.angularjs.org/1.5.8/$injector/modulerr?p0=ceptecafeApp&p1=Error%3A%20%5B%24injector%3Amodulerr%5D%20Failed%20to%20instantiate%20module%20infinite-scroll%20due%20to%3A%0AError%3A%20%5B%24injector%3Anomod%5D%20Module%20'infinite-scroll'%20is%20not%20available!%20You%20either%20misspelled%20the%20module%20name%20or%20forgot%20to%20load%20it.%20If%20registering%20a%20module%20ensure%20that%20you%20specify%20the%20dependencies%20as%20the%20second%20argument.%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fnomod%3Fp0%3Dinfinite-scroll%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28233%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6380%0A%20%20%20%20at%20n%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A5853)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A6165%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15513%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15530%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24injector%2Fmodulerr%3Fp0%3Dinfinite-scroll%26p1%3DError%253A%2520%255B%2524injector%253Anomod%255D%2520Module%2520'infinite-scroll'%2520is%2520not%2520available!%2520You%2520either%2520misspelled%2520the%2520module%2520name%2520or%2520forgot%2520to%2520load%2520it.%2520If%2520registering%2520a%2520module%2520ensure%2520that%2520you%2520specify%2520the%2520dependencies%2520as%2520the%2520second%2520argument.%250Ahttp%253A%252F%252Ferrors.angularjs.org%252F1.5.8%252F%2524injector%252Fnomod%253Fp0%253Dinfinite-scroll%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A4%253A28233%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A6380%250A%2520%2520%2520%2520at%2520n%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A5853)%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A6165%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15513%250A%2520%2520%2520%2520at%2520r%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A4%253A28681)%250A%2520%2520%2520%2520at%2520f%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15361)%250A%2520%2520%2520%2520at%2520http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15530%250A%2520%2520%2520%2520at%2520r%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A4%253A28681)%250A%2520%2520%2520%2520at%2520f%2520(http%253A%252F%252F52.23.175.251%252Fapp%252Fvendor-a935ac298f.js%253A5%253A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28233%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15789%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15530%0A%20%20%20%20at%20r%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A4%3A28681)%0A%20%20%20%20at%20f%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A15361)%0A%20%20%20%20at%20rt%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A17290)%0A%20%20%20%20at%20a%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A3923)%0A%20%20%20%20at%20ie%20(http%3A%2F%2F52.23.175.251%2Fapp%2Fvendor-a935ac298f.js%3A5%3A4233)
    at http://52.23.175.251/app/vendor-a935ac298f.js:4:28233
    at http://52.23.175.251/app/vendor-a935ac298f.js:5:15789
    at r (http://52.23.175.251/app/vendor-a935ac298f.js:4:28681)
    at f (http://52.23.175.251/app/vendor-a935ac298f.js:5:15361)
    at rt (http://52.23.175.251/app/vendor-a935ac298f.js:5:17290)
    at a (http://52.23.175.251/app/vendor-a935ac298f.js:5:3923)
    at ie (http://52.23.175.251/app/vendor-a935ac298f.js:5:4233)
    at re (http://52.23.175.251/app/vendor-a935ac298f.js:5:3472)
    at HTMLDocument.<anonymous> (http://52.23.175.251/app/vendor-a935ac298f.js:10:4045)
    at p (http://52.23.175.251/app/vendor-a935ac298f.js:2:8692)
- Please follow the issue template below for bug reports and feature requests.
- Tickets opened without any of these informations will be **closed** without any explanation.

**Comment by pascalgrimaud** [comment]
How can we help you if we have no information for trying to reproduce the issue?
Read and answer all questions in the template

**Comment by sonersivri** [comment]
I solved problem but it is a critical error.

**Comment by jdubois** [comment]
Then please @sonersivri -> give us a full bug report, so we can review the issue and solve it.
And if you have a solution, then that's even better, please submit it.
This is community project we do on our free time, and none of us is ready to spend his Sunday evening on an incomplete bug report, because that's really not fun.

**Comment by jeffrey-anderson** [comment]
I have the same exact problem.  I upgraded to 4.0.2 but it's still broken.  As indicated in the top comment, I had to remove 'infinite-scroll', 'angular-loading-bar' from ./src/main/webapp/app/app.module.js.  Note for me the problem was both dev and prod profile.  It works fine on the dev box but when I checkin to git and checkout on a different box, the problem happens.  I will reproduce with a brand new app this evening time permitting.

BTW - love jHipster!
